src/lib/DailyTodosCard/TodosCard.svelte

```diff
@@ -15,7 +15,7 @@
 	{:else}
 		<p>No todos added</p>
 	{/if}
-	<CallToAction href="/add-todos">Add {dailyTodos.length ? 'more' : ''} todos</CallToAction>
+	<CallToAction id="add-todos" href="/add-todos">Add {dailyTodos.length ? 'more' : ''} todos</CallToAction>
```

Comment:
I think instead of making it mandatory to add an ID I'll just make it optional to add it (kinda like how you did it with the newTab = false)